assets/scss/templates/_header.scss

```diff
@@ -90,7 +93,8 @@
 
   li {
     margin-right: 25px;
-    padding: 10px 10px 10px 0;
+    padding: 0 10px 0 0;
```

Comment:
is it easier put `padding-right: 10px` not?

Reply:
Maybe @damianmuti wants to ensure 0 padding.